doc/nrf/ug_creating_matter_accessory.rst

```diff
@@ -0,0 +1,274 @@
+In this example value is updated randomly, but in a real sensor application it would be updated with a value obtained from external measurement.  
+
+
+Don't forget to declare these handler functions in :file:`src/app_task.h` for a proper build.
+
+| Now add cases for new events in ``AppTask::DispatchEvent``:
+.. code-block:: C++
+
+        void AppTask::DispatchEvent(const AppEvent &event)
+        {
+                switch (event.Type) {
+                case AppEvent::FunctionPress:
+                        FunctionPressHandler();
+                        break;
+                case AppEvent::FunctionRelease:
+                        FunctionReleaseHandler();
+                        break;
+                case AppEvent::FunctionTimer:
+                        FunctionTimerEventHandler();
+                        break;	
+                case AppEvent::SensorActivate:
+                        SensorActivateHandler();
+                        break;
+                case AppEvent::SensorDeactivate:
+                        SensorDeactivateHandler();
+                        break;
+                case AppEvent::SensorMeasure:
+                        SensorMeasureHandler();
+                        break;
+                default:
+                        LOG_INF("Unknown event received");
+                        break;
+                }
+        }
+        
+Create a callback for sensor activation/deactivation
+****************************************************
+
+Up until now events ``SensorActivate`` and ``SensorDeactivate`` were never added to the event queue.   
+This is because attribute of the OnOff cluster is never changed from within ``AppTask``. 
+It can only be changed externally, for example through Matter controller.
+
+
+However, sensor application needs to know how this attribute is changing, so there is a need to implement a callback function to post one of these events every time OnOff attribute changes.
+To see a list of callback functions that are customizable one can open :file:`src/gen/callback-stub.cpp` and look for functions with ``__attribute__((weak))``.      
+It is sufficient just to implement ``emberAfPostAttributeChangeCallback`` (read the description in :file:`src/gen/callback-stub.cpp`).
+
+
+In order to do it, create a new file, for example :file:`src/zcl_callbacks.cpp` and implement a callback inside it.
+A simple exemplary implementation might look like this:     
+
+.. code-block:: C++
+
+        #include "app_task.h"
+        #include <app/util/af.h>
+
+        using namespace ::chip;
+
+        void emberAfPostAttributeChangeCallback(EndpointId endpoint, ClusterId clusterId, AttributeId attributeId, uint8_t mask,
+                                                uint16_t manufacturerCode, uint8_t type, uint16_t size, uint8_t *value)
+        {
```

Comment:
Return if the modified cluster is not on/off or the modified attribute is not on/off?